parallaxinc/Parallax-IDE
# Create and update example project on launch

#### What's this PR do?

This PR adds an examples folder with source files on launch. This will automatically add new files if they exist in the examples directory but are not in the user's example project, allowing new files to be added later.
#### How should this be tested by the reviewer?

Launch the app and verify that the Examples project exists and is populated. Add a new example and rebuild, verify that it is added after the fact.
#### What are the relevant tickets?

Closes #295
